.travis.yml

```diff
@@ -1,11 +1,13 @@
 language: python
-sudo: false
+dist: xenial
+sudo: true
```

Comment:
Why is this required?

Reply:
See https://github.com/travis-ci/travis-ci/issues/9815

I hope, we can remove this change soon.